drivers/interrupt_controller/intc_gicv3.c

```diff
@@ -299,6 +299,16 @@ static void gicv3_rdist_enable(mem_addr_t rdist)
 		return;
 	}
 
+	if (GICR_IIDR_PRODUCT_ID_GET(sys_read32(rdist + GICR_IIDR)) >= 0x2) {
```

Comment:
It would be more readable if we had an annotation here. Just a suggestion, not mandatory.